product_docs/docs/pem/9/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx

```diff
@@ -19,7 +19,7 @@ You must configure the PEM database server to work with PgBouncer. This example
 2.  Create a user named pem_admin1 (not a superuser) with `pem_admin` and `pem_agent_pool role` membership on the PEM database server:
 
     ```ini
-    pem=# CREATE USER pem_admin1 PASSWORD 'ANY_PASSWORD' LOGIN
+    pem=# CREATE USER pem_admin1 PASSWORD 'ANY_PASSWORD' LOGIN CREATEROLE;
     CREATEROLE;
```

Comment:
```suggestion
    pem=# CREATE USER pem_admin1 PASSWORD 'ANY_PASSWORD' LOGIN CREATEROLE;
```
The additional CREATEROLE permission is already there, it's just got a line break before it for some reason. Need to remove the spurious line.